scripts/ci/check_lang_names.py

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env python3
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import sys
+import xml.etree.ElementTree as etree
+
+
+if __name__ == "__main__":
+    script_path = os.path.dirname(os.path.abspath(__file__))
+
+    ###
+    # 1. Grab the list of expected languages in the i18n submodule
+
+    i18n_path = os.path.join(script_path, '..', '..', '3rdparty', 'i18n')
+    if not os.path.exists(i18n_path):
+        print(f"i18n submodule not found in path {i18n_path}")
+        sys.exit(1)
```

Comment:
You can [exit directly with a message](https://docs.python.org/3/library/sys.html#sys.exit)

```
        sys.exit(f"i18n submodule not found in path {i18n_path}")
```